src/lib/y2network/widgets/bond_port.rb

```diff
@@ -196,15 +196,23 @@ def justify_dev_name(name)
       # device names if at least two devices shared the same physical port id
       # TODO: backend method
       #
-      # @param slaves [Array<String>] bonding slaves
-      # @return [Hash{String => Array<String>}] of duplicated physical port ids
-      def repeated_physical_port_ids(slaves)
+      # NOTE: term port is slightly overloaded here. Name of method refers to
+      # physical ports of a NIC card (one card can have multiple "plugs" - ports).
+      # On the other hand param name refers to pure virtual bonding ports (network
+      # devices provided by the system which are virtualy tighted together into a
+      # virtual bond device)
+      #
+      # @param b_ports [Array<String>] bond ports = devices included in the bond
+      # @return [Hash{String => Array<String>}]
+      #   maps physical ports to non-singleton arrays of bond ports
+      def repeated_physical_port_ids(b_ports)
         physical_port_ids = {}
 
-        slaves.each do |slave|
-          if physical_port_id?(slave)
-            p = physical_port_ids[physical_port_id(slave)] ||= []
-            p << slave
+        b_ports.each do |b_port|
+          if physical_port_id?(b_port)
+            p_port = physical_port_id(b_port)
```

Comment:
@mchf I have changed this method and the following one to make the code+comments clearer